src/SeleniumTestCase.php

```diff
@@ -3,51 +3,65 @@
 namespace Modelizer\Selenium;
 
 use Modelizer\Selenium\Services\Application as Laravel;
+use Modelizer\Selenium\Services\InteractsWithPage as Interactions;
+use Modelizer\Selenium\Services\WaitsForElements;
+use Modelizer\Selenium\Services\WorksWithDatabases;
 use PHPUnit_Extensions_Selenium2TestCase;
 
 class SeleniumTestCase extends PHPUnit_Extensions_Selenium2TestCase
 {
-    use Laravel;
+    use Laravel,
+        Interactions,
+        WorksWithDatabases,
+        WaitsForElements;
+
+    /**
+     * @var string
+     */
+    protected $baseUrl;
 
     protected function setUp()
     {
         $this->setUpLaravel();
-
-        $this->setBrowserUrl(env('APP_URL', 'http://localhost/'));
+        $this->baseUrl = env('APP_URL', 'http://localhost/');
+        $this->setBrowserUrl($this->baseUrl);
         $this->setBrowser(env('DEFAULT_BROWSER', 'chrome'));
     }
 
-    protected function visit($path)
-    {
-        $this->url($path);
-
-        return $this;
-    }
-
-    protected function see($text, $tag = 'html')
+    public function setupPage()
     {
-        $this->assertContains($text, $this->byTag($tag)->text());
-
-        return $this;
+        if (!empty(getenv('SELENIUM_WIDTH')) && !empty(getenv('SELENIUM_HEIGHT')) &&
```

Comment:
Are you saying this so you can change the window's width and hight during a test?

Otherwise we could just have it be a property on a sub-class so you can test different widths and heights for different types of tests, say between mobile/tablet/desktop.

And then in the setup method set the width and the height if it's present.